bitnami/rabbitmq-cluster-operator/Chart.yaml

```
@@ -28,4 +28,4 @@ maintainers:
 name: rabbitmq-cluster-operator
 sources:
   - https://github.com/bitnami/charts/tree/main/bitnami/rabbitmq-cluster-operator
-version: 3.4.3
+version: 3.4.4
```

I think the chart version should be minor because the change in the crds is minor too.

```suggestion
version: 3.5.0
```